docker-compose.yml

```diff
@@ -18,7 +18,7 @@ services:
       - POSTGRES_USER=${DATABASE_USERNAME}
       - POSTGRES_DB=${DATABASE}
     ports:
-      - "5432:5432"
+      - "5433:5432"
```

Comment:
Have to put this for my local development, but i always change it to [5432] when I'm resolving confilcts.